lib/cylc/option_parsers.py

```diff
@@ -32,12 +32,16 @@ class CylcOptionParser(OptionParser):
     """Common options for all cylc CLI commands."""
 
     MULTITASK_USAGE = """
-TASKID is a pattern to match task proxies or task families, or groups of them:
+TASKID is a pattern to match task proxies, or families, or groups of them:
 * [CYCLE-POINT-GLOB/]TASK-NAME-GLOB[:TASK-STATE]
 * [CYCLE-POINT-GLOB/]FAMILY-NAME-GLOB[:TASK-STATE]
 * TASK-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
 * FAMILY-NAME-GLOB[.CYCLE-POINT-GLOB][:TASK-STATE]
 
+WARNING: this command matches and operates on task proxy instances in the
+scheduler task pool, NOT abstract tasks in the workflow. If a task is not
```

Comment:
Maybe this instead (for this branch), to remain consistent with Cylc 7 terminology where we have not generalised to 'workflow'?

```suggestion
scheduler task pool, NOT abstract tasks in the suite. If a task is not
```

Reply:
Fair enough!